src/appleseed/renderer/modeling/project-builtin/cornellboxproject.cpp

```diff
@@ -31,6 +31,9 @@
 #include "cornellboxproject.h"
 
 // appleseed.renderer headers.
+#include "renderer/api/environment.h"
```

Comment:
`renderer/api/*` files are for users of the appleseed library (such as appleseed.cli, appleseed.studio, all the command line tools, and users of the C++ SDK). Within appleseed itself you should include only the required headers.

Reply:
@dictoon; in case they are removed functions like _ConstantEnvironmentEDFFactory()_ and _ConstantEnvironmentEDFFactory()_
are becoming out of scope. What libraries are required to be included other that these to make use of above functions?

Reply:
Replace by `renderer/modeling/environmentedf/constantenvironmentedf.h` or something